umd/base/utils.py

```diff
@@ -147,6 +147,12 @@ def create(self, trusted_ca_dir=None):
                         % (self.domain_comp_country,
                            self.domain_comp)])
 
+    def get_cert_subject(self, hostname):
```

Comment:
`issue_cert` method returns a `OwnCACert` object that has already an attribute `subject`. See below (in `umd/products/gram5.py`)